docs/node-guide/quick-start.md

```diff
@@ -3,18 +3,31 @@ sidebar_position: 1
 ---
 
 # Quick start with GaiaNet Node
+This guide provides the requisite knowledge necessary to quickly get started with installing a GaiaNet node. 
 
-First, let's install GaiaNet a one-line installer on Mac, Linux, or Windows WSL. This installer will download the latest version of GaiaNet Node. Learn more about [system requirements](system-requirements).
+### Prerequisites
+Before you get started, ensure that you have the following on your system:
+
+| System | Minimum Requirements |
+|---|---|
+| OSX with Apple Silicon (M1-M4 chip) | 16GB RAM (32GB recommended) |
+| Ubuntu Linux 20.04 with Nvidia CUDA 12 SDK | 8GB VRAM on GPU |
+| Azure/AWS | Nvidia T4 GPU Instance |
+
```

Comment:
The above three systems can't cover all the systems that we support.  I suggest that we use RAM as the limit.